tailwind.config.js

```diff
@@ -32,5 +60,5 @@ module.exports = {
   variants: {
     extend: {}
   },
-  plugins: []
+  plugins: [require('@tailwindcss/aspect-ratio')]
```

Comment:
to use this plugin its required install it using yarn

https://github.com/tailwindlabs/tailwindcss-aspect-ratio

just run `yarn add @tailwindcss/aspect-ratio` and submit te PR again

Reply:
we dont need this plugin as this wont work for safari .... i forgot to remove this plugin. Will remove this.
Thanks @vczb

Reply:
great! approved! thanks